packages/patterns-react/src/patterns/CardArray/CardArray.js

```diff
@@ -28,46 +30,38 @@ const CardArray = ({ title, content }) => {
   const containerRef = useRef();
 
   useEffect(() => {
-    setCardsHeight();
+    setSameHeight();
+    window.addEventListener('resize', setSameHeight);
   });
 
   /**
-   * Set the cards to have the same height as the bigger one
+   * Function that activates the sameheight utility
    */
-  const setCardsHeight = () => {
-    let biggest = 0;
-    const cards = Array.prototype.slice.call(containerRef.current.children);
-    cards.forEach(card => {
-      if (card.offsetHeight > biggest) {
-        biggest = card.offsetHeight;
-      }
-    });
-    cards.forEach(card => {
-      card.style.height = biggest + 'px';
-    });
+  const setSameHeight = () => {
+    sameheight(
+      containerRef.current.getElementsByClassName(
+        `${prefix}--card-link__title`
+      ),
+      'md'
+    );
+    sameheight(containerRef.current.children, 'md');
   };
 
   return featureFlag(
     DDS_CARD_ARRAY,
-    <section
-      className={`${prefix}--cardarray`}
-      data-autoid={`${stablePrefix}--cardarray`}>
-      <div className={`${prefix}--cardarray__container`}>
-        <div className={`${prefix}--cardarray__row`}>
-          <div className={`${prefix}--cardarray__col`}>
-            <h3 className={`${prefix}--cardarray__title`}>{title}</h3>
-          </div>
+    <section data-autoid={`${stablePrefix}--cardarray`}>
+      <ContentGroup heading={{ copy: title }}>
```

Comment:
@RaphaelAmadeu and I talked a little about this earlier this week. My comment here is similar to the https://github.com/carbon-design-system/ibm-dotcom-library/pull/882#discussion_r361229946 I gave to Sann earlier in the week as well. We've wrapped this `ContentGroup` with a `section` element. `ContentGroup` in a way is the containing component. That's what we're building here, and that's what `"CardArray"` is. It's a `ContentGroup`. Does the CardArray need a wrapper just so it can have a stable selector, is its stable selector technically shared with the `ContentGroup`, or is it completely a separate thing? It's almost as if we are saying from a hierarchical standpoint that `CardArray` is greater than `ContentGroup` when they are both one in the same.

@jeffchew maybe this is something we can discuss in our next huddle? I'm going to add it for now.